commands.js

```diff
@@ -2007,6 +2007,7 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (target === 'show') return this.parse('/showblacklist');
```

Comment:
Also:
[23:09] +panpawn: can you uh do me a favor
[23:10] +panpawn: can you check if user "show" is abandoned or not?
[23:11] &Trickster: it is